src/Resources/views/Form/form_admin_fields.html.twig

```diff
@@ -35,7 +35,7 @@ file that was distributed with this source code.
 
 {% block form_help %}
     {% apply spaceless %}
-        <span class="help-block sonata-ba-field-widget-help">{{ parent() }}</span>
+        <span class="help-block sonata-ba-field-widget-help sonata-ba-field-help">{{ parent() }}</span>
```

Comment:
I added this class because is [the one used in the deprecated `help`](https://github.com/sonata-project/SonataAdminBundle/blob/3.x/src/Resources/views/Form/form_admin_fields.html.twig#L404).